src/components/sbb-toggle-check/sbb-toggle-check.tsx

```diff
@@ -85,10 +92,12 @@ export class SbbToggleCheck implements AccessibilityProperties {
           aria-describedby={this.accessibilityDescribedby}
           aria-labelledby={this.accessibilityLabelledby}
         />
-        <span class="toggle-check__container">
-          <slot />
-          <span class="toggle-check__slider">
-            <span class="toggle-check__circle">
+        <span class="sbb-toggle-check__container">
+          <span class="sbb-toggle-check__label" hidden={!this._hasLabelText}>
```

Comment:
was a little bit surprised how it's going to be rendered, because i thought it will end up in hidden=false, which would be wrong for a boolean attribute. But it seems to be rendered to correct way, so if false, hidden is not there at all :-)

Reply:
Double-checked in all desktop browsers, Safari included, seems to work fine 🚀

Reply:
I think this is already done be tsx somehow so it's browser independent :-)